src/metabase/sync/fetch_metadata.clj

```diff
@@ -58,7 +57,7 @@
   (let [tables (sync-util/db->reducible-sync-tables database :schema-names schema-names :table-names table-names)]
     (eduction
      (mapcat (fn [table]
-               (for [x (table-fields-metadata database (t2.realize/realize table))]
+               (for [x (table-fields-metadata database table)]
```

Comment:
`t2.realize/realize` is called in `db->reducible-sync-tables` instead, so `table` is already realized.